src/main/java/world/inetum/realdolmen/inetumrealJobs/payload/request/LoginRequest.java

```diff
@@ -1,25 +1,27 @@
 package world.inetum.realdolmen.inetumrealJobs.payload.request;
 
+import javax.validation.constraints.Email;
 import javax.validation.constraints.NotNull;
 
 public class LoginRequest {
 
-    private String username;
+    @Email
```

Comment:
Also annotate NotNull (or NotBlank, but that's partially covered by the Email annotation)

Reply:
Fixed